guides/source/action_mailbox_basics.md

```diff
@@ -380,3 +412,16 @@ end
 ```
 
 Please refer to the [ActionMailbox::TestHelper API](https://api.rubyonrails.org/classes/ActionMailbox/TestHelper.html) for further test helper methods.
+
+## Incineration of InboundEmails
+
+By default, an [`InboundEmail`][] that has been processed will be incinerated after 30 days. The InboundEmail is considered as processed when its status changes to `delivered`, `failed` or `bounced`.
```

Comment:
Maybe we should link to InboundEmail on its first appearance under "What is Action Mailbox?" instead?